scanner/datastore/postgres/vuln_update.go

```diff
@@ -3,48 +3,81 @@ package postgres
 import (
 	"context"
 	"errors"
-	"fmt"
-	"net/http"
-	"strings"
 	"time"
 
 	"github.com/jackc/pgx/v4"
 )
 
-const lastVulnUpdateKey = `last-vuln-update`
-
-// GetLastVulnerabilityUpdate retrieves the last vulnerability update from the database.
+// GetLastVulnerabilityUpdate implements MatcherMetadataStore.GetLastVulnerabilityUpdate.
 //
-// The returned time will be in the form of http.TimeFormat.
+// Assumes the last update is the oldest update timestamp in the table.
```

Comment:
I don't think I understand this query. Should _last update_ be the most recent timestamp in previous round of update under the multi-bundle update mode?

Reply:
> Should last update be the most recent timestamp for one specific updater name?

Previously, a single timestamp was used to represent the entire bundle, which contained multiple vulnerabilities. This change uses multiple timestamps, with each updater having its own timestamp within the bundle. As a result, the oldest timestamp from all the updaters in the bundle is now equivalent to the single timestamp that was used to represent the entire bundle in the previous logic.